examples/cache/1_cache.py

```diff
@@ -0,0 +1,47 @@
+from dff.core.engine.conditions import true
+from dff.core.engine.core import Context, Actor
+from dff.core.engine.core.keywords import TRANSITIONS, RESPONSE
+from dff.core.engine.labels import repeat
+from dff.core.pipeline import Pipeline
+from dff.script.utils.singleton_turn_caching import cache
+from dff.utils.testing.common import check_happy_path, is_interactive_mode, run_interactive_mode
+
+
+external_data = {"counter": 0}
+
+
+@cache
+def cached_response(_):
```

Comment:
Can we add more comments, that for same `args` function return same for one turn. Need more comments and user guided docs.